### DISCUSSION TITLE
No maps to download

### DISCUSSION THREAD

**Discussion by Beerforfree1** [osmandapp/OsmAnd#19672] [General]
For some unknown reason my Osmand App stopped to show maps for download under MAPS & RESOURCES. I've tried downloading the latest release 4.6.14, I've uninstalled and reinstalled it with no changes at all.
I'm using a Motorola G4+ with Android 7

**Comment by dmpr0** [comment]
Hi! 
The problem is related to the security certificates used by OsmAnd, starting from 2024 they are not supported on Android versions prior to 7.1.

The issue has been fixed in version 4.7.10, which is available on Google Play.

**Reply by wpee** [reply]
hello thanks that works.
BUT.... I have an OSMAND+ paid subscription, and I'm on Android 6.
I know the releases directory at OSMAND.NET but as for as I know these are als OSMAND releases. I tried downloading version 4.7.10, installed it and it worked. Tn so for that I can download maps again, BUT only limited. Resetting my subscriptions apparently does not work for converting OSMAND to OSMAND+ (or I did something wrong)
As said I'm looking to find the OSMAND+ releases ?
(and I need a working version for android 6, is this also 4.7.10 ?)

**Comment by Beerforfree1** [comment]
Thanks buddy
